examples/nft-d/nft.go

```diff
@@ -0,0 +1,390 @@
+/*
+Package nft_d contains divisible non-fungible NEP-11-compatible token
+implementation. This token can be minted with GAS transfer to contract address,
+it will retrieve company name from the transfer data and produce NFT which
+represents shares of the specified company.
```

Comment:
Shares are so naturally fungible, that we might consider using something else. Although that might be a problem of divisible NFTs in general, each of them could easily be proper NEP-17, but somehow they're NFTs. Usually I'm thinking of some real estate in this case, each object is unique, there are lots of them (unlike publicly traded companies) and each can be owned by a set of owners. Not sure it makes for a good example.

Reply:
Yes, I think it's a good example, but possibly it has the same problem as shares, because if you share a part of a flat (e.g. 10m² out of 30m²), then you still can't locate the particular part of this flat and say that it's 10m² of a kitchen or 10m² of a bathroom, you just own 1/3 of a flat which is also naturally fungible.

And also I remember the https://github.com/neo-project/proposals/pull/130#discussion_r591114136, but looks like it's a problem of all NEP11-divisible tokens.

Reply:
Fixed.

Reply:
BTW, just discussed --- imagine a NeoFS object, a perfect NFT. You can theoretically give access to it only to its owner, but what if you want to give access to a number of people? That's where divisible NFTs might be handy, you give 0.00001 part of this NFT to the user and he can access it now.

Reply:
Fixed.